src/users/list/v2.rs

```diff
@@ -0,0 +1,108 @@
+//! [GET /_synapse/admin/v2/users/:user_id](https://github.com/matrix-org/synapse/blob/master/docs/admin_api/user_admin_api.rst#list-accountshttps://github.com/matrix-org/synapse/blob/master/docs/admin_api/user_admin_api.rst#query-user-account)
+
+use crate::serde::boolean_as_uint;
+use js_int::UInt;
+use ruma::api::ruma_api;
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Clone, Debug)]
+pub struct UserMinorDetails {
+    /// The user's name.
+    pub name: String,
+
+    /// todo: doc but I do not know what this is
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub user_type: Option<String>,
+
+    /// Is the account a guest
+    #[serde(with = "boolean_as_uint")]
+    pub is_guest: bool,
+
+    /// Is the user a server admin
+    #[serde(with = "boolean_as_uint")]
+    pub admin: bool,
+
+    /// Is the account deactivated
+    #[serde(with = "boolean_as_uint")]
+    pub deactivated: bool,
+
+    /// The user's display name, if set.
+    pub displayname: String,
+
+    /// The user's avatar URL, if set.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub avatar_url: Option<String>,
+}
+
+ruma_api! {
+    metadata: {
+        description: "list accounts endpoint",
+        method: GET,
+        name: "list_accounts",
+        path: "/_synapse/admin/v2/users",
+        rate_limited: false,
+        authentication: AccessToken,
+    }
+
+    #[derive(Default)]
+    request: {
+        /// TODO: this should be treated as opaque, i.e. a newtype, so that only values returned from responses can be used here
+        /// Offset in the returned list. Defaults to 0.
+        #[serde(skip_serializing_if="Option::is_none")]
```

Comment:
There should be spaces here:

```suggestion
        #[serde(skip_serializing_if = "Option::is_none")]
```

Same in the other attributes.